### ISSUE TITLE
Noticeable "flash" when screens load

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#664]
The use of JS to load "desktop" CSS causes the font to resize in many common desktop scenarios. This causes a noticeable flash, which, whilst not exactly a WCAG 2 failure is distracting enough to be problematic.
